scripts/tasks/download.js

```diff
@@ -1,12 +1,13 @@
 'use strict'
 
-var github = require('octonode')
-var client = github.client()
-var evRepo = client.repo('nodejs/evangelism')
+const github = require('octonode')
+const https = require('https')
+const path = require('path')
+const fs = require('fs')
 
-var https = require('https')
-var path = require('path')
-var fs = require('fs')
+const basePath = path.join(__dirname, '..', '..', 'locale', 'en', 'blog', 'weekly-updates')
+const client = github.client()
+const evRepo = client.repo('nodejs/evangelism')
```

Comment:
Since you are refactoring, you can directly create evRepo without those intermediate requires.